common/src/main/java/com/wynntils/models/items/ItemModel.java

```diff
@@ -102,6 +102,15 @@ public <T extends WynnItem> Optional<T> asWynnItem(ItemStack itemStack, Class<T>
         return Optional.of((T) wynnItem);
     }
 
+    public <T> Optional<T> asWynnItemPropery(ItemStack itemStack, Class<T> clazz) {
```

Comment:
I missed this :) Great addition. We should use this over the old version in all places possible (as a follow up PR)

Reply:
I fixed it everywhere, actually, as part of this PR. Maybe a bit messy, but, well... 🤷